generator-prisma-client/nuxt4-starter-nodejs/.env.example

```
@@ -0,0 +1,2 @@
+DATABASE_URL="prisma+postgres://..."
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)